services/ui-src/src/styles/_header.scss

```
@@ -69,23 +68,22 @@
       &.user-email-button {
         font-weight: $font-weight-bold;
         margin: 0;
-        padding: 0;
-        padding-right: ($padding * 3);
         position: relative;
-        text-align: center;
+        margin-left: ($margin * 10);
 
         @media only screen and (min-width: 768px) {
           text-align: right;
         }
 
         a {
+          padding-right: ($padding * 3);
```

I noticed that when you click this menu the tabbed focus styles change the background color to white. That makes sense if you have dark text, but if you light text on a dark background like this it kinda does the opposite of accessibility. Slightly changing that tabbed focus style will fix it up. 

Note: I've made a similar suggestion for the menu options below.
```suggestion
        a {
          &:focus {
            background-color: initial;
          }
          padding-right: ($padding * 3);
```